### PR TITLE
Update mobilenet_v3.py

### PR DESCRIPTION
**Pull request by BlainWu**
Hi, I found that normalization operation is included in the definition of mobilenetv3 model and Mobile netv2 doesn't do that.
I don't think include normalization in the model a good idea, it should be outside the model. First of all, it limits the ability to customize. Secondly, I find that the normalization of 255 directly results in the accuracy of the model almost reduced to random after the model is converted to tflite int-8 version. 
So I think it's better to remove it in the model.

### CODE REVIEW

**Comment by BlainWu** [comment]
@googlebot I signed it!

**Review by qlzh727 — CHANGES_REQUESTED** [review]
Thanks for sending the PR.

There are some history for having this rescaling layer here. Historically, we have the preprocessing API for each model like tf.keras.applications.mobilenet_vx.preprocess_input(). The implementation of that method is to rescale the image and add an offset. Usually, it is very easy for end user to forget including that method in their code, which result into very bad result. So we change our approach in mobilenet v3, moving that preprocessing into the model, and make the body preprocess_input() to be empty.

The current PR will just revert our intended change, and produce poor result if user didn't preprocess the inputs.
